content/akeneo-web-api.yaml

```diff
@@ -2157,6 +2291,61 @@ definitions:
     allOf:
       - $ref:  '#/definitions/ItemList'
       - $ref: '#/definitions/AttributeOption'
+  AttributeGroup:
+    type: object
```

Comment:
Missing "required: ["code"]" that indicates that the "code" property is required